src/components/input/Input.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+
+import Input from './Input'
+
+describe('<Input/>', () => {
+  it('matches snapshot', () => {
+    const wrapper = shallow(<Input />)
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  it('should invoke onChange when triggered', () => {
+    const onChangeSpy = jest.fn()
+    const component = mount(<Input onChange={onChangeSpy} />)
+    component.find('input').simulate('change')
+    expect(onChangeSpy).toHaveBeenCalledTimes(1)
+  })
+
+  it('should have correct id, required, type props and display label', () => {
+    const id = 'InputId'
+    const label = 'Input field'
+    const type = 'text'
+    const component = mount(
+      <Input id={id} label={label} type={type} required />
+    )
+    expect(component.find('input').prop('id')).toBe(id)
+    expect(component.find('input').prop('required')).toBe(true)
+    expect(component.find('input').prop('type')).toBe(type)
+    expect(component.find('label').text()).toBe(label)
+  })
```

Comment:
This isn't testing any business logic, so can just be done as part of the snapshot.

Reply:
👍 I've removed this.

Reply:
Apologies, I wasn't too clear with that one. What I meant was to move the `id` and other prop passing to the snapshot test so that they may be captured within the snapshot render. This removed test can remain deleted.

Reply:
👍 I think I've done this.